ninjalooter/working/group-templates.txt

```diff
@@ -0,0 +1,53 @@
+Tank group (1-2)
```

Comment:
Is this file actually used or just for documentation purposes? If it's a doc, I'd rename it to `.rst` and format it as a real Restructured Text docfile